doc/sphinx/practical-tools/utilities.rst

```diff
@@ -515,6 +515,11 @@ To build, say, two targets foo.vo and bar.vo in parallel one can use
   + ``-extra-phony`` and ``-extra`` are deprecated. To provide additional target
     (``.PHONY`` or not) please use ``CoqMakefile.local``.
 
+.. note::
+
+  Due to limitations with the compilation chain, makefiles generated
+  by ``coq_makefile`` won't correctly compile OCaml plugins with OCaml
```

Comment:
```suggestion
  by ``coq_makefile`` won't compile correctly OCaml plugins with OCaml
```

"correctly compile" is not idiomatic.

Reply:
"correctly compile" sounds more idomatic to my ear.  "compile OCaml plugins correctly" also sounds fine to my ear, and both of these sound better than "compile correctly OCaml plugins"

Reply:
Right.  My bad.